### ISSUE TITLE
you dont actually have to construct a spacepod properly

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#26994]
(WEB REPORT BY: rabbit2rabbit REMOTE: 206.221.180.138:7777)
# Revision
2ffc595a3d9e9bb15b338df7e66f010fa41d9171

# Description

spacepod construction is broken
you don't have to use any of the parts except the armor and frame to build it
i dont know exactly how this works as i just saw someone else do it but it seems to be pretty common knowledge and people aren't bugreporting it

**Comment by SysadminJeroen** [comment]
```
[08:42:28] Player PM from-Rabbit2Rabbit/(Phat Dan): add a comment to it saying that badguy knows the steps to circumvent spacepod construction so ask them for more details bcuz i couldnt report on it properly

```

**Comment by Exxion** [comment]
Duplicate of #25736

**Comment by DeityLink** [comment]
Fixed by #27519
